### PR TITLE
claws-mail: update to 4.3.0

### PR DESCRIPTION
**Pull request by androidnisse**
**Summary**
- Release notes can be found [here](https://www.claws-mail.org/news.php).

**Test Plan**
- Installed and setup an account.

**Checklist**

- [X] Package was built and tested against unstable

**Packaging notes**
- Appstream metadata added from flathub.

### CODE REVIEW

File: packages/c/claws-mail/files/org.claws_mail.Claws-Mail.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+      <li>Graceful, and sophisticated interface</li>
+      <li>Easy configuration, intuitive operation</li>
+      <li>Abundant features</li>
+      <li>Extensibility</li>
+      <li>Robustness and stability</li>
+    </ul>
+    <p>The appearance and interface are designed to be familiar to new users coming from other popular email clients, as well as experienced users. Almost all commands are accessible with the keyboard.</p>
+    <p>The messages are managed in the standard MH format, which features fast access and data security. You'll be able to import your emails from almost any other email client, and export them just as easily.</p>
+    <p>Lots of extra functionality, like an RSS aggregator, calendar, or laptop LED handling, are provided by extra plugins.</p>
+    <p>This is an unofficial package. (See package metadata for details.)</p>
```
Comment (malfisya) [inline packages/c/claws-mail/files/org.claws_mail.Claws-Mail.appdata.xml:25]: Remove this paragraph as no to confuse user

File: packages/c/claws-mail/files/org.claws_mail.Claws-Mail.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+  </description>
+
+  <icon type="remote" height="128" width="128">https://github.com/flathub/org.claws_mail.Claws-Mail/raw/master/share/icons/hicolor/128x128/apps/claws-mail.png</icon>
+  <icon type="remote" height="64" width="64">https://github.com/flathub/org.claws_mail.Claws-Mail/raw/master/share/icons/hicolor/64x64/apps/claws-mail.png</icon>
+  <icon type="remote" height="48" width="48">https://github.com/flathub/org.claws_mail.Claws-Mail/raw/master/share/icons/hicolor/48x48/apps/claws-mail.png</icon>
```
Comment (malfisya) [inline packages/c/claws-mail/files/org.claws_mail.Claws-Mail.appdata.xml:30]: Remove all remote icons, we use what the applications ship

File: packages/c/claws-mail/files/org.claws_mail.Claws-Mail.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+          <li>Several memory leaks</li>
+          <li>Verify_folderlist_xml() for fresh starts</li>
+          <li>Printf formats for size_t and goffset arguments</li>
+          <li>Alertpanel API use in win32 part of mimeview.c</li>
+          <li>Pid handling in debug output of kill_children_cb()</li>
+          <li>Incorrect pointer arithmetic in w32_filesel.c</li>
+        </ul>
+      </description>
+    </release>
+  </releases>
```
Comment (malfisya) [inline packages/c/claws-mail/files/org.claws_mail.Claws-Mail.appdata.xml:353]: Either add new entry or remove all release component. We ships newer version of the package.

File: packages/c/claws-mail/files/org.claws_mail.Claws-Mail.appdata.xml [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+
+  <content_rating type="oars-1.0">
+    <content_attribute id="social-contacts">intense</content_attribute>
+  </content_rating>
+  <update_contact>danny_AT_dannyvanheumen.nl</update_contact>
```
Comment (malfisya) [inline packages/c/claws-mail/files/org.claws_mail.Claws-Mail.appdata.xml:358]: Remove update contact, we don't want user to contact the dev directly if they need support.

**Review by malfisya — CHANGES_REQUESTED** [review]
Makes sure to verify metainfo **again** with `appstream-builder --packages-dir=. --include-failed -v` after applying those changes

**Comment by androidnisse** [comment]
> Makes sure to verify metainfo with `appstream-builder --packages-dir=. --include-failed -v` after applying those changes

That is done. 👍

**Review by malfisya — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**